Templates/SupportTests.stencil

```diff
@@ -0,0 +1,8 @@
+import Foundation
```

Comment:
What about giving the `Templates` directory a structure that represents the templates?
```
Templates/
  default/
  swiftui/
```
Rather than placing all the files under `Templates/`? That way one can spot which files belong to which template.

Reply:
That's how it works. Files under `Templates` are shared between `default` and `swiftui`

Reply:
What about:

```
Templates/
  shared/
  default/
  swiftui/
```

Reply:
That will create a problem since a folder is directly translated to a template. I can modify this, but I don't like the fact that then it's a little bit more ambiguous what's a template and what's not (you will not be able to see that just by looking at that directory). Users should put shared templates in `ProjectDescriptionHelpers`, this is just a workaround for tuist built-in ones.